spec/session_spec.coffee

```diff
@@ -508,6 +509,32 @@ describe 'Session', ->
 
             expect(@biskoto.get(@basic_meta_cookie_name)).to.deep.equal metadata
 
+        context 'and shop_cookie_policy is specified as basic', ->
+          beforeEach ->
+            @shop_cookie_policy = 'basic'
+
+          it 'sets the session first-party basic cookie value', ->
+            metadata = { app_type: @metadata.app_type, cp: 'b', tags: @metadata.tags }
+            @stub_params({ session: @analytics_session, metadata: metadata })
+
+            @init()
```

Comment:
Here we need to pass an optional parameter `shop_cookie_policy` and by default this needs to be `full`. The init method is defined in L299